src/axom/mir/detail/elvira_detail.hpp

```diff
@@ -821,6 +823,96 @@ class TopologyBuilder<ExecSpace, CoordsetView, TopologyView, MatsetView, PHShape
   View m_view;
 };
 
+//------------------------------------------------------------------------------
+/*!
+ * \brief Base template for letting us pick different behaviors for making clean zones.
+ *
+ * \tparam ExecSpace The execution space where the algorithm will run.
+ * \tparam TopologyView The view type that describes the Blueprint input mesh.
+ * \tparam CoordsetView The view type that describes the coordinates.
+ * \tparam MatsetView The view type that describes matset.
+ * \tparam NDIMS The dimension of the topology.
+ */
+template <typename ExecSpace, typename TopologyView, typename CoordsetView, typename MatsetView, int NDIMS>
+struct MakeCleanOutput
```

Comment:
Consider calling this `MakeCleanMesh`.  "Output" doesn't carry much information.

Reply:
I'll address this in the next PR.